make/conf/jib-profiles.js

```diff
@@ -1272,7 +1274,7 @@ var getJibProfilesDependencies = function (input, common) {
 
         libffi: {
             organization: common.organization,
-            module: "libffi-" + input.build_platform,
+            module: "libffi-" + input.target_platform,
```

Comment:
This change is needed to select to right artifact when building x86 on a x64 build machine.